docs/running-workflows/launch-with-badge/index.md

```diff
@@ -0,0 +1,74 @@
+# Launch workflow using a badge
+
+To make it easier to launch a workflow, we provide a "Launch-on-REANA" badge that allows running a workflow with a single click.
+
+For example, to launch a [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit) from GitHub on REANA instance at CERN, you can click on the badge below:
+
+[![Launch ROOT6 workflow from GitHub](https://www.reana.io/static/img/badges/launch-on-reana.svg)](https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit)
```

Comment:
I feel like it will be too much. We have the main launch page with a nice plot for this.

Reply:
Adding one picture next to the badge (that we already have) won't be too much... it'll simply improve the visualisation.

It'll be useful for people who come to this page whilst reading docs. If they see only badge, it is hard to visualise what it does. If they see the plot and the text, it's easy to visualise what it does.

(Recall also that people won't be going to `reana.cern.ch/launch` much, since we don't advertise this URL much.... It's mostly the other way round, we point here from that launcher page.)

Reply:
Should this plot be the same as we have on the `launch/` page for demos? Or it should be a screenshot of what the `launch/` page looks like? Cause I think I got confused (plot vs screenshot).

Reply:
Yes, just the picture of the plot as we have it on the launcher page.  You can center it and place it just above "Launch on REANA@CERN" badge. Very easy and small change; just to make the page look more attractive.

Reply:
Done. Added a picture of the example plot for RooFit.